File: Java/linear_search.java

public class linear_search{

    public static void main(String[] args){
        int[] arr = {21, 11, 24, 12, 35, 13, 83, 14};
        System.out.println(linearSearch(arr, 83));

    }

    static int linearSearch(int[] arr, int target){
        int len = arr.length;
        if(len == 0){
            return -1;
        }
        for(int i = 0; i<len; i++){
            if(arr[i] == target){
                return i;
            }
        }
        return-1;
    }
}

File: Python/linear_search.py
def linear_search(nums, target):
    if(len(nums)==0):
        return -1
    else:
        for i in range(len(nums)):
            if(nums[i] == target):
                return i
    return -1

arr = [23, 18, 20, 38, 97]
print(linear_search(arr, 97))